python/sqlalchemy/src/rest-service.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+import logging
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker, relationship
+from http.server import HTTPServer, BaseHTTPRequestHandler
+import json
+from io import BytesIO
+import jsonpickle
+from model import User, Order, OrderLine, Product
+import config as cfg
+from data_access_util import DataAccessUtil
+from sqlalchemy import create_engine
+from sqlalchemy import MetaData
+from model import Base
+import sys
+import traceback
+
+
+logging.basicConfig(
```

Comment:
Lets move this to the config if possible, and change default level to higher than debug - say INFO level?

Reply:
Done